components/org.wso2.carbon.identity.auth.valve/src/main/java/org/wso2/carbon/identity/auth/valve/util/APIErrorResponseHandler.java

```diff
@@ -202,4 +213,18 @@ private static boolean isCorrelationIDPresent() {
 
         return MDC.get(CORRELATION_ID_MDC) != null;
     }
+
+    private static String getRealmInfo() {
+
+        return "Basic realm=" + ServerConfiguration.getInstance().getFirstProperty("HostName");
```

Comment:
For a tenant or an organization is this the correct value to show?

Reply:
For a tenant, the tenant name could be shown as the realm as per [1].
However it also mentions if no realm is mentioned, clients often show the hostname.
This behaviour is consistant with authorization error from other components as well.

[1] https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/WWW-Authenticate